app/components/test/cookies_banner_component.rb

```diff
@@ -0,0 +1,7 @@
+class Test::CookiesBannerComponent < ViewComponent::Base
```

Comment:
Can we go with AbTest instead please? Makes it clearer what these components are about imo